src/logic/graphDisassembling.ts

```diff
@@ -27,8 +27,9 @@ export function getEdgesArray(graph: Graph): Array<Edge> {
 
   for (let from = 0; from < graph.length; from++) {
     for (let to = 0; to < graph[from].length; to++) {
-      const edge = new Edge(from, to);
-      if (edge.from !== edge.to && !isDuplicate(edge)) {
+      const cost = graph[from][to];
+      const edge = new Edge(from, to, String(cost));
+      if (edge.from !== edge.to && !isDuplicate(edge) && cost > 0) {
```

Comment:
May there be cost <= 0 ?

Reply:
Yes there can be cost 0. Means edge is non-existant